src/test_util.rs

```diff
@@ -412,6 +428,15 @@ impl TransactionDependencyProvider for Context {
             "header not found".to_string(),
         ))
     }
+
+    fn get_block_extension(
+        &self,
+        _block_hash: &Byte32,
+    ) -> Result<Option<ckb_types::packed::Bytes>, TransactionDependencyError> {
+        Err(TransactionDependencyError::NotFound(
+            "block extension not found".to_string(),
+        ))
```

Comment:
What's the difference between returning `Ok(None)` and an error?

Reply:
I unintentionally overlooked the impact of returning an error in this code, and I simply want to ensure that the code compiles without any errors.

Do you think we should return `Ok(None)` here?

Reply:
I'm getting it now. Error should be returned if the block is not found, and None if the extension is not found, right?

Reply:
Yes. 

And the code you commented is used by unit test.

In non-test code, `get_block_extension` has been implemented:

https://github.com/nervosnetwork/ckb-sdk-rust/blob/8bd9d3c6c84fdf901fe2f92f5e499f9a88a51fe2/src/traits/mod.rs#L162-L169